### PR TITLE
Remove the 'DaCe Daemon not running' notification when not optimizing

### LINKED ISSUE #12: "Daemon not working" popup still appears when Terminal mode is default

### PR DESCRIPTION
**Pull request by phschaad**
This closes #12.

Additionally adds a code of conduct and authors list file.

### CODE REVIEW

File: src/daceInterface.ts [hunk 1]
```diff
@@ -497,6 +498,12 @@ implements MessageReceiverInterface {
     public promptStartDaemon() {
         if (this.daemonBooting)
             return;
+
+        // If the optimization panel isn't open, we don't want to interact
+        // with the daemon. Don't prompt in that case.
+        if (!OptimizationPanel.getInstance().isVisible())
```
Comment (tbennun) [inline src/daceInterface.ts:504]: I'd say not just the prompt, but also the whole daemon should not start until the optimization panel is visible

**Review by tbennun — CHANGES_REQUESTED** [review]
minor behavioral change but other than that, very good work! we can use it for showing the other panes later on. Also thanks for the author list and code of conduct

File: src/daceInterface.ts (hunk 1 above)
Comment (phschaad) [reply src/daceInterface.ts:504]: The way it is right now, it actually shouldn't do that - the daemon is only called once one of the optimization panels get activated. I added a sanity check to make sure it doesn't happen regardless

**Review by tbennun — APPROVED** [review]

File: src/daceInterface.ts (hunk 1 above)
Comment (tbennun) [reply src/daceInterface.ts:504]: Thanks

**Comment by phschaad** [comment]
Thanks for the review!

**Pull request merged**